src/lib.rs

```diff
@@ -110,6 +77,57 @@ extern crate serde_derive;
 #[macro_use]
 mod macros;
 
+// ############################################################################
+// Public API
+// ############################################################################
+pub use self::{
+    chain::quorum_count,
+    error::{InterfaceError, RoutingError},
+    event::{ClientEvent, ConnectEvent, Event},
+    event_stream::EventStream,
+    id::{FullId, P2pNode, PublicId},
+    node::{Node, NodeBuilder},
+    pause::PausedState,
+    quic_p2p::{Config as NetworkConfig, NodeInfo as ConnectionInfo},
+    xor_space::{Prefix, XorName, XorNameFromHexError, Xorable, XOR_NAME_BITS, XOR_NAME_LEN},
```

Comment:
seems only Prefix and XorName is required (in vault at least)

Reply:
OK, but I would maybe also expose `XOR_NAME_LEN`, which I think is common practice for types that publicly wrap a fixed-length array ([example](https://docs.rs/ed25519-dalek/1.0.0-pre.3/ed25519_dalek/constant.PUBLIC_KEY_LENGTH.html)).

Reply:
I would agree in this case as it fits a common pattern in rust :+1:  I was going for the keep as much private as possible /within reason but in his case I agree